### PR TITLE
[OCDM] Adapter gstreamer version compile fixes

### PR DESCRIPTION
**Pull request by pawel-stanek-ml**

### CODE REVIEW

File: Source/ocdm/adapter/gstreamer/open_cdm_adapter.cpp [hunk 1]
```diff
@@ -4,6 +4,21 @@
 #include <gst/gst.h>
 #include <gst/base/gstbytereader.h>
 
+inline bool mappedBuffer(GstBuffer *buffer, bool writable, uint8_t **data, uint32_t *size)
```
Comment (pawel-stanek-ml) [inline Source/ocdm/adapter/gstreamer/open_cdm_adapter.cpp]: Note this is copy&paste from SVP version but I came to the conclusion it's not worth creating a separate header and source files  just for this.

**Comment by modeveci** [comment]
It is ok, my bad I could not test on default gstreamer impl. Thanks

File: Source/ocdm/adapter/gstreamer/open_cdm_adapter.cpp (hunk 1 above)
Comment (modeveci) [reply Source/ocdm/adapter/gstreamer/open_cdm_adapter.cpp]: It is ok, it is already an inline func.

**Comment by pawel-stanek-ml** [comment]
No problem at all.
Will you merge it or I should?

**Pull request merged**